packages/wrangler/src/dev/local.tsx

```diff
@@ -187,7 +207,7 @@ function useLocalWorker({
       });
 
       local.current.stdout?.on("data", (data: Buffer) => {
-        console.log(`${data.toString()}`);
+        process.stdout.write(data);
```

Comment:
as recommended in https://github.com/cloudflare/wrangler2/pull/746#issuecomment-1086612408

Reply:
On downside to doing this is that I am not sure how this plays with InkJS patching `console`...?
An alternative is just to `trimEnd()` the data...

Reply:
iiuc it's actually react patching console, and only in dev mode. I'm happy to replace with trimEnd, no biggy. behaviour-wise it was fine